e2e/ansible/playbooks/hyperconverged/test-replica-node-affinity/replica-node-affinity.yaml

```diff
@@ -148,27 +148,27 @@
          until: "'patched' in patch_out.stdout"
          delay: 20
          retries: 5
-     
+
        - name: 8) Get pod name
          shell: kubectl get pods --all-namespaces -o wide | grep "{{ pv_name.stdout }}" | grep rep | grep "{{ node1.stdout }}" | awk '{print $2}'
          args:
            executable: /bin/bash
-         register: pod_name 
+         register: pod_name
 
        - name: 8a) Delete the replica on one of the nodes
-         shell: kubectl delete pod "{{ pod_name.stdout }}" --grace-period=0 --force
+         command: kubectl delete pod "{{ pod_name.stdout }}" --grace-period=0 --force
```

Comment:
Does the linter recommend replacing shell with command ? If so, what is the criteria ?

Reply:
Thanks for asking,
Yes Linter recommended to replace shell.
 The shell module is potentially more dangerous compare to command module (ok, nothing is really stopping you doing command: rm -rf --no-preserve-root) and should only be used when you actually need shell functionality. 
If you’re not stringing two commands together (using pipes or even just && or ;), you don’t really need the shell module. Similarly, expanding shell variables or file globs require the shell module. If you’re not using these features, don’t use the shell module. If you are using these features, think twice if you can rewrite the shell command to make it more Ansible friendly.

Reply:
In the CI, we are sourcing the profile before running each kubectl command. Can you confirm if the current playbook can run successfully in the CI env ?